alpha/apps/kaltura/modules/extwidget/actions/serveFlavorAction.class.php

```diff
@@ -186,14 +207,16 @@ protected function servePlaylist($entry, $captionLanguages)
 		}
 		
 		list($entryIds, $durations, $referenceEntry, $captionFiles) = myPlaylistUtils::executeStitchedPlaylist($entry, $captionLanguages);
+		list($offsets, $durations) = $this->retrievePlaylistAttributesData($entry, $durations);
```

Comment:
is it intended to support also the existing stitched playlist? else i would have preffer not to have th possibility to override duration for existing use case with the new function

Reply:
yes, it's intended to support stitched playlist